src/components/colorbar/draw.js

```diff
@@ -512,7 +512,7 @@ function drawColorBar(g, opts, gd) {
             .data(fillLevels);
         fills.enter().append('rect')
             .classed(cn.cbfill, true)
-            .style('stroke', 'none');
+            .attr('style', '');
```

Comment:
Well, it's tricky a bit to explain.. The first thing to note is that when calling `Plotly.newPlot()`, the colorbar is drawn twice, ie. the code above runs twice (which might be a bug (?) as the first draw seems correct, but I didn't dig much into that). When calling `Plotly.react()` though, it runs once. 

Now, we have : 
```js
var fills = g.select('.' + cn.cbfills)
    .selectAll('rect.' + cn.cbfill)
    .attr('style', '')        // -> existing <rect> have empty `style` attribute
    .data(fillLevels);
fills.enter().append('rect')
    .classed(cn.cbfill, true)
    .style('stroke', 'none'); // -> missing <rect> are added with `style='stroke: none;'`
    fills.exit().remove();
```

So, if I'm not wrong :
- After calling `Plotly.newPlot()`, because the code runs twice, there is no missing `<rect>` on the second pass obviously, and in the end all of them have an empty `style` attribute (for contour colorbar, each element get colored via the `fill` attribute) 
- After calling `Plotly.react()` however, because the code runs once, those `<rect>` that were newly added, if any, will have in the end `style='stroke: none;'` 

This discrepancy causes this [test case](https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/colorbar_test.js#L517-L551) to fail, because some attributes are changing between transition/redraw. So this commit is just to fix that. 

Now the question is : how the heck this test case hasn't ever failed until this PR ? 

I think this could have to do with the double call and possible race conditions.. Hopefully I've been able to reproduce the failure without the colorbar fix, by using a mock with more data than those used previously by the test (longer to draw, hence the suspicion about race conditions). I'm going to commit that so you can double check all of this.